librec/src/test/java/librec/util/GammaTest.java

```diff
@@ -0,0 +1,56 @@
+package librec.util;
+
+import librec.util.Gamma;
+import org.junit.Assert;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+import java.lang.reflect.Method;
+
+public class GammaTest {
+
+  @Rule public ExpectedException thrown = ExpectedException.none();
+
+  /* testedClasses: Gamma */
+
+  /*
+   * Test generated by Diffblue Deeptest.
+   * This test case covers:
+   *  - conditional line 68 branch to line 69
+   */
+
+  @Test
+  public void digammaInputNegativeInfinityOutputNaN() {
```

Comment:
Java Double.NEGATIVE_INFINITY != Double.NaN by default. That makes this an interesting and quite possibly useful test case!